src/i18n/en.js

```diff
@@ -6,4 +6,7 @@ export default {
   'search-media-info': '$1 selected',
   'wikimedia-commons': 'Commons',
   'wikipedia': 'Wikipedia',
+  'btn-publish': 'Publish',
+  'btn-publish-wikistory': 'Publish Wikistory',
+  'tags-not-set': 'Not set yet'
```

Comment:
@stephanebisson can you do the `fr` translation for these?

Reply:
Done